src/Elasticsearch.Net.Aws/Elasticsearch.Net.Aws/AwsSettings.cs

```diff
@@ -24,5 +24,11 @@ public class AwsSettings
         /// </summary>
         public string SecretKey { get; set; }
 
+        /// <summary>
+        /// Gets or sets the security token
+        ///  Required.
+        /// </summary>
+        public string Token { get; set; }
```

Comment:
we shouldn't need to expose this since it's only necessary to provide it when using instance profile credentials. Instance profile credentials will be used when no access key or sercret are provided.